docs/en/operation/Troubleshooting.md

```diff
@@ -29,10 +29,15 @@ Workers are doing, especially when running into any issues. If you do not unders
 search for them in the [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users),
 in the case the problem has been discussed before. 
 You can also join our [Slack channel](https://slackin.alluxio.io/) and seek help there.
+You can find more details about the Alluxio server logs [here]({{ '/en/operation/Basic-Logging.html#server-logs' | relativize_url }}).
 
 The client-side logs are also helpful when Alluxio service is running but the client cannot connect to the servers.
 Alluxio client emits logging messages through log4j, so the location of the logs is determined by the client side
 log4j configuration used by the application.
+You can find more details about the client-side logs [here]({{ '/en/operation/Basic-Logging.html#application-logs' | relativize_url }}).
+
+The user logs in `${ALLUXIO_HOME}/logs/user/` are the logs from running Alluxio shell.
```

Comment:
the user logs can also be users using other native APIs or FileSystem APIs? not just Alluxio shell?

Reply:
I believe with FileSystem API, the user will define their own logger logging to their own specified location. The FileSystem code API based code should count as "client application".

In our code `logs/user/user_*.log` is only used in ALLUXIO_USER_JAVA_OPTS, which seems to be specified for shell as https://github.com/Alluxio/alluxio/blob/6b35ea2d826dd79b5c9c60ab4d5317e8bdc49a74/conf/alluxio-env.sh.template#L53